src/handlers/authorizations/{ID}.js

```diff
@@ -46,7 +46,7 @@ module.exports = {
    * produces: application/json
    * responses: 202, 400, 401, 403, 404, 405, 406, 501, 503
    */
-  get: async (request, h) => {
+  get: async (c, request, h) => {
```

Comment:
here second parameter is `request` and some places its `req` is it on purpose?

Reply:
Will standardize all